dist/main.js

```diff
@@ -0,0 +1,35 @@
+const manager = new WeatherManager
+const renderer = new Renderer
+
+const loadPage = async function(){
+    const dataArr = await manager.getDataFromDB()
+    renderer.loadCities(dataArr)  //manager.getData()
+}
+loadPage()
+
+
+const searchCity = async function(){
+    let cityName = $("#input").val()
+    $("#input").val("")
+
+    const citiesArr = await manager.getCityData(cityName)
+    renderer.loadCities(citiesArr)
+}
+
+
+$('#citiesContainer').on('click', 'i', async function(){
+            if($(this).closest('div').attr('class') === "false"){
+                let cityName = $(this).closest('.city').find('.name').text()
+                const citiesArr = await manager.saveCity(cityName)
+                renderer.loadCities(citiesArr)
+            }
+})
+
+
+$('#citiesContainer').on('click', 'i',async function(){
+            if($(this).closest('div').attr('class') === "true"){
```

Comment:
you can just say 
`if($(this).closest('div').attr('class'))`
it is the same thing